### PR TITLE
GOB-981 Levels to BRK SJT secure types. Add level to compare

### PR DESCRIPTION
**Pull request by JKoetsier**

### CODE REVIEW

File: gobcore/typesystem/__init__.py [hunk 1]
```diff
@@ -181,11 +181,11 @@ def get_modifications(entity, data, model):     # noqa: C901
 
     for field_name, field in model.items():
         gob_type = get_gob_type(field['type'])
-        old_value = gob_type.from_value(getattr(entity, field_name))
+        old_value = gob_type.from_value(getattr(entity, field_name), level=field.get('level'))
```
Comment (Vegter) [inline gobcore/typesystem/__init__.py]: why is this needed?

Comment (JKoetsier) [reply gobcore/typesystem/__init__.py]: As discussed. Resolved.
